tests/goldfile.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
I would put the goldfile in the same directory as the other code - tests/integration-tests/performance

Reply:
I will do this